lib/cli.js

```diff
@@ -1,15 +1,15 @@
 #!/usr/bin/env node
-'use strict';
 
 /**
  * CLI entry for SFMC DevTools
  */
 
-const Util = require('./util/util');
-const yargs = require('yargs');
-const Mcdev = require('./index');
+import { Util } from './util/util.js';
 
-yargs
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
```

Comment:
whats that?

Reply:
yargs doesnt like es6 modules 1:1, this is the workaround provided  - https://github.com/yargs/yargs/issues/1854#issuecomment-787509517